### DISCUSSION TITLE
First Impressions

### DISCUSSION THREAD

**Discussion by ng-druid** [Green-Software-Foundation/discussions#28] [Introductions]
Hello,

I'm Todd the founder of a recent charity dedicated to building open source web solutions freed from scope 1,2, and 3 carbon emissions. Last year our charity contributed a climate aware proxy to CarbonHack that received the most actionable award. We have since built upon our CarbonHack entry transforming it into a climate aware database HEDGE. We are currently building our open source climate aware CMS named Spearhead. My preferred pronouns are (he/him). Our charity is a US registered 501(c)3 located in Newark Delaware. Anyone that would like to connect can send a friend request on LinkedIn. More information about use can be found on our website. Happy to be part of the green tech movement. Always looking for ways to make software more environment friendly.

* Charity: https://rollthecloud.com/
* HEDGE: https://github.com/rollthecloudinc/hedge
* Spearhead: https://github.com/rollthecloudinc/spearhead
* LinkedIn: https://www.linkedin.com/in/toddzmijewski/

Thanks

**Comment by NAMRATA-WOKE** [comment]
Very cool @ng-druid. Thanks for sharing Sprearhead!  I'll include it in this week's newsletter.

**Comment by NAMRATA-WOKE** [comment]
@ng-druid Since there is more activity in the other [Welcome Discussion](https://github.com/orgs/Green-Software-Foundation/discussions/2), you might want to reintroduce yourself there!
Don't forget to hit the subscribe button to get notified of new activity!
